overlay/ports/qt5-base/CONTROL

```diff
@@ -0,0 +1,5 @@
+Source: qt5-base
+Version: 5.12.4-6

```

Comment:
We are using Qt 5.12.3 in Mixxx 2.3. Hopefully there are no regressions with 5.12.4, but we'll have to test...

Reply:
Please bump the version to 5.12.4-7 to distinguish it from the upstream vcpkg version.

Reply:
done